extensions/twilio/v2/actions/sendSmsDuringBusinessHours/sendSmsDuringBusinessHours.ts

```diff
@@ -0,0 +1,121 @@
+import { z, ZodError } from 'zod'
+import { fromZodError } from 'zod-validation-error'
+import twilioSdk from '../../../common/sdk/twilio'
+import { type Action } from '@awell-health/extensions-core'
+import { type settings } from '../../../settings'
+import { Category, validate } from '@awell-health/extensions-core'
+import { SettingsValidationSchema } from '../../../settings'
+import { FieldsValidationSchema, fields, dataPoints } from './config'
+import { isNil } from 'lodash'
+import {
+  getNextDateWithinBusinessHours,
+  isDateBetweenBusinessHours,
+} from '../../../../../src/utils/getNextDateWithinBusinessHours'
+
+export const sendSmsDuringBusinessHours: Action<
+  typeof fields,
+  typeof settings
+> = {
+  key: 'sendSmsDuringBusinessHours',
+  title: 'Send SMS during business hours',
+  description:
+    'Send a text message during business hours from a given telephone number to a recipient of your choice.',
+  category: Category.COMMUNICATION,
+  fields,
+  dataPoints,
+  previewable: false,
+  onActivityCreated: async (payload, onComplete, onError) => {
+    try {
+      const {
+        settings: {
+          accountSid,
+          authToken,
+          messagingServiceSid: defaultMessagingServiceSid,
+        },
+        fields: { recipient, message, messagingServiceSid, timeZone },
+      } = validate({
+        schema: z
+          .object({
+            settings: SettingsValidationSchema,
+            fields: FieldsValidationSchema,
+          })
+          .superRefine((value, ctx) => {
+            // if both `from` values missing - throw error
+            if (
+              isNil(value.settings.messagingServiceSid) &&
+              isNil(value.fields.messagingServiceSid)
+            ) {
+              ctx.addIssue({
+                code: z.ZodIssueCode.custom,
+                fatal: true,
+                message:
+                  '"Messaging Service SID" is missing in both settings and in the action field.',
+              })
+            }
+          }),
+        payload,
+      })
+
+      const client = twilioSdk(accountSid, authToken, {
+        region: 'IE1',
+        accountSid,
+      })
+
+      const nowString = new Date().toISOString() // curent datetime in UTC
+      const isBetweenBusinessHours = isDateBetweenBusinessHours(
+        nowString, // UTC date
+        timeZone // offset, target timezone
+      )
```

Comment:
```suggestion
      const now = new Date()
      const isBetweenBusinessHours = isDateBetweenBusinessHours(
        now,
        timeZone // offset, target timezone
      )
```

See the related comment in `isDateBetweenBusinessHours`. The conversion to string format is not needed.